### PR TITLE
Add new getters in time-interval

### PR DESCRIPTION
**Pull request by jcarrillosrc**

### CODE REVIEW

File: src/TimeInterval/TimeInterval.php [hunk 1]
```diff
@@ -89,6 +89,31 @@ public function getInSeconds()
         return (int) \ceil($this->milliseconds / 1000);
     }
 
+    /**
+     * @return int
+     */
+    public function getInDays(): int
+    {
+        return (int) \ceil(
```
Comment (mastercoding) [inline src/TimeInterval/TimeInterval.php]: floor

File: src/TimeInterval/TimeInterval.php [hunk 2]
```diff
@@ -89,6 +89,31 @@ public function getInSeconds()
         return (int) \ceil($this->milliseconds / 1000);
```
Comment (mastercoding) [inline src/TimeInterval/TimeInterval.php]: floor

File: src/ReadableTimeInterval/ReadableTimeIntervalGenerator.php [hunk 3]
```diff
@@ -61,6 +62,10 @@ public function getReadableTimeText(int $secondsToTransform)
             $textComponents[] = $this->translator->transChoice('timeInterval.seconds', $seconds, ['%count%' => $seconds]);
         }
 
+        if( true === empty($textComponents) ){
```
Comment (mastercoding) [inline src/ReadableTimeInterval/ReadableTimeIntervalGenerator.php:65]: isn't this the last check from 61?

Comment (jcarrillosrc) [reply src/ReadableTimeInterval/ReadableTimeIntervalGenerator.php:65]: Returned value was '{space}and{space}' when zero was received

**Pull request merged**